app/frontend/entrypoints/application.scss

```
@@ -32,6 +32,7 @@ $container-max-widths: (
 @import "~/styles/chapters.scss";
 @import "~/styles/colors.scss";
 @import "~/styles/courses.scss";
+@import "~/styles/grade_scheme_summary.scss";
```

Stylelint (17.14.1)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)